### DISCUSSION TITLE
"Non-interaction" on Custom Events

### DISCUSSION THREAD

**Discussion by sptv2reg** [plausible/analytics#2729] [Anything Else]
Hi

If a custom event is triggered after pageview, the session will be marked as non-bounce:

https://github.com/plausible/analytics/discussions/863#discussioncomment-516481

I wonder if we could mark custom events as "non-interaction" like in Google Analytics. In many of my use cases it doesn't really make sense to let them determine bounce/non-bounce.

**Comment by metmarkosaric** [comment]
thanks for the feedback @sptv2reg! makes sense and something we should add in the future. there's a thread already on this idea here: https://github.com/plausible/analytics/discussions/1914
